wiki/Active-Effect-Guide.md

```diff
@@ -0,0 +1,442 @@
+# FoundryVTT dnd5e Active Effects Examples
+
+![Up to date as of 2.3.x](https://img.shields.io/badge/dnd5e-v2.3.x-informational)
+
+This document only covers Active Effects available to the Core dnd5e System.
+
+## Legend
+
+`[number]` - These square brackets mean "replace this with your value of the type within the brackets".
+So this example: `+[number]` would mean you input `+3`.
+
+`[formula]` - When `formula` is mentioned in this document it means this value can be populated with any dice formula. For example, Bless adds several effects with the Effect Value of `1d4`.
+
+As part of this, an [Actor's Rolldata](https://github.com/foundryvtt/dnd5e/wiki/Roll-Formulas) is available as "@attributes." Useful examples:
```

Comment:
We probably need another note here then, according to [this](https://github.com/foundryvtt/dnd5e/pull/2482#discussion_r1365998926) discussion, warning that supplying rolldata is not supported for all active effects, and it's kind of arbitrary which ones it is supported for. Not sure how best to explain that either.

Reply:
I think that anywhere a die formula is supported, a rolldata path will be too?

Reply:
> I think that anywhere a die formula is supported, a rolldata path will be too?

If it supports dice, then it's a string that is evaluated when needed (at least fully, some parts are sometimes evaluated immediately for sheets), and it supports roll data.

The other direction is not always true, some strings support roll data (for example `attributes.hp.bonuses`), but not dice.